docs/technical_reference/costing/ion_exchange.rst

```diff
@@ -6,54 +6,172 @@ Costing Method Parameters
 
 The following parameters are constructed for the unit on the FlowsheetCostingBlock (e.g., `m.fs.costing.ion_exchange`) when applying the `cost_ion_exchange` costing method in the ``watertap_costing_package``:
 
-.. csv-table::
-   :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "parameter_name", "1", ":math:`\text{dimensionless}`"
+.. csv-table::
+   :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units", "Notes"
+
+   "Anion exchange resin cost", ":math:`c_{res}`", "``anion_exchange_resin_cost``", "205", ":math:`\text{USD2020 }\text{ft}^{-3}`", "Assumes strong base polystyrenic gel-type Type II. From EPA-WBS cost model."
```

Comment:
Sorry to be so nitpicky about the units, but they should look like `":math:`\text{USD}_{2012}\text{/ft}^3`"`. This makes it so that the year is in the subscript and a fraction is used rather than the negative exponent

Reply:
Ah was lost on me that this was to make it a subscript so that makes more sense.

Reply:
Not sure if you're handling this piecewise or not, but the negative exponents should also be changed to fractions. And there are also still a few "$" in the "Costing Metho Variables" that should be changed to USD_YYYY

Reply:
The "$" in the "Costing Method Variables" would be converted to the ``base_currency`` that is assigned at the costing package, so there isn't really a year to assign... I see other unit model docs do have USD_XXXX, so I guess they calculate in USD_XXXX and then convert to ``base_currency`` elsewhere, but that isn't strictly true for the IX implementation (e.g., displaying the value for ``capital_cost_resin`` would be in units of ``base_currency``, not USD_2020).  I suppose I could do something like USD_base? If we want the USD units in the docs to reflect what would be the displayed result for a variable, I would have to change the IX costing model. I am not opposed to this but just want to be clear.

I had converted the fractions to negative exponents to be consistent with other docs... but looking around now I guess there isn't real consistency in that regard for our docs so I can convert it back.

Reply:
Ah, the "$" is probably fine then. Or maybe we should just put USD instead. As for the fractions vs negative exponents, I _think_ all of the costing documentation uses fractions, so we can at least keep that consistent, if nothing else.